### DISCUSSION TITLE
"Offline" as "Off" Feature - Implemented or Possible?

### DISCUSSION THREAD

**Discussion by Tom5640** [milo526/homebridge-tuya-web#538] [Ideas]
Hey @milo526 ,

Loving the plugin - I have some Tuya lights that I control with a switch, and they are only smart so that they allow me to change the colour etc when I would like to.  Some of my plugins have a toggle to be able to show "offline as off" within HomeKit, meaning when the device is switched off at the switch (not connected to power), rather than "No Response" being shown within HomeKit, the device just shows as "Off".  Is this something I can configure within the json or could be implemented?
Tom

**Comment by milo526** [comment]
This feature is not currently available; it is requested in #441.
